contrib/vdirsyncer.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+    <!-- click requires a proper UTF-8 locale -->
+    <key>EnvironmentVariables</key>
+    <dict>
+           <key>LANG</key>
+           <string>en_US.UTF-8</string>
+           <key>LC_ALL</key>
+           <string>en_US.UTF-8</string>
```

Comment:
I now did a bit of research on this and indeed there doesn't seem to be a way to do this the way I imagined (dynamically setting the correct locale at runtime). I don't really care about the value itself, but it should be made clear to the reader that this part may have to be changed to the correct locale string.